README.md

```diff
@@ -83,7 +83,9 @@ and filled.
 | REDIS_VERSION         | Version of the Redis image |
 | POSTGRES_VERSION      | Version of the Postgres image |
 | NGINX_VERSION         | Version of the NGINX image |
-| UWSGI_PROCESSES       | Number of uwsgi processes to start (default 10) |
+| WEB_SERVER            | Web server for the CDCS (e.g. `uwsgi`, `gunicorn`)
+| PROCESSES             | Number of uwsgi processes (default `--processes=8`) / gunicorn workers to start (default `workers=cpu_count() * 2 + 1`) |
+| THREADS               | Number of uwsgi/gunicorn threads (default 8)|
```

Comment:
The description of `THREADS` could be improved by explaining that number of threads is per process/worker